MAINTAINERS.md

```diff
@@ -1,6 +1,22 @@
 # Maintainers
 This file contains a list of the maintainers of tldr-pages.
 
+
+## Current collaborators
+The following people are collaborators on this repository, enabling them to label and close issues,
+and review and merge pull requests.
```

Comment:
It would be nice to add the link to the automated list: [https://github.com/tldr-pages/tldr/settings/collaboration](https://github.com/tldr-pages/tldr/settings/collaboration)